website/blog/_guide/content-process.md

```diff
@@ -30,17 +30,59 @@ The file name should be in the format `YYYY-MM-DD-title-in-kebab-case-with-dashe
 where `YYYY-MM-DD` is the date of the post, and `title-in-kebab-case-with-dashes` is a descriptive title of the post.
 Please do not underscores or spaces in the file names, directory names, or slugs because URL paths typically use dashes.
 
+### Tags
+
+Tags are an important part of every blog post, and they appear at the top of the front matter.
```

Comment:
I moved that out of our common writing guide because tags and keywords are blog-specific